src/_includes/site_navigation.html

```diff
@@ -9,21 +9,21 @@
         {% for section in sections_by_title %}
           <li class="related__section-link">
```

Comment:
Is now the time to rename the sass classes too? Perhaps to `site-navigation` and `site-navigation__`

Reply:
Yep, good idea! I've updated the relevant commit - do you think the new names make sense?

Reply:
(Also changed the `related_order` property to `navigation_order`

Reply:
Looks perfect! Yes and navigation_order makes sense to me too :)